src/Tracking/DisasterTrackerApp.Dal/DisasterTrackerContext.cs

```diff
@@ -34,20 +36,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         modelBuilder.Entity<CalendarEvent>()
             .HasIndex(e => e.GoogleEventId)
             .IsUnique(true);
-        
-        
+
         modelBuilder.Entity<DisasterEvent>()
             .HasKey(e => e.Id);
```

Comment:
i recomend to left unique index
        modelBuilder.Entity<DisasterEvent>()
            .HasIndex(e => e.ExternalApiId)
            .IsUnique(true);